### PR TITLE
#77 with ocurl

### PR DESCRIPTION
**Pull request by dinosaure**

### CODE REVIEW

**Comment by dinosaure** [comment]
God, it's green!

File: syndic.opam [hunk 1]
```diff
@@ -18,15 +18,24 @@ build: [
...
-  "tls" {with-test}
+  "fmt" {with-test}
+  "ocurl" {with-test}
+  "fpath" {with-test}
+  "ocplib-json-typed" {with-test}
+  "base-unix" {with-test}
+  "jsonm" {with-test}
 ]
+
+extra-source "test/001.feed" { src: "http://16andcounting.libsyn.com/rss" }
```
Comment (samoht) [inline syndic.opam]: is this downloaded even when the tests don't run? I am not sure that's the expected behaviour /cc @AltGr

Comment (AltGr) [reply syndic.opam]: Indeed, this will always be downloaded. The `extra-source` section doesn't allow filters at the moment...

Comment (dinosaure) [reply syndic.opam]: :(

**Pull request merged**